### PR TITLE
Add python FastAPI template with Docker and Web Apps deployment

### PR DESCRIPTION
**Pull request by shanepeckham**
## Purpose

* ...
This template adds a starter for a FastAPI power skill template with uvicorn production web server, a Dockerfile to build the solution and a terraform script to deploy to Web Apps for linux for a secure API connection. The API includes authentication and scaffolding to simply add custom logic.

## Does this introduce a breaking change?

```
[ ] Yes
[ X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```
Starter template for python

## How to Test
*  Get the code
* Add functionality
* Run locally:

```bash
uvicorn app:app --reload --port 5000
```
* Build docker image, push to a container registry and deploy via terraform scripts to Azure Web Apps

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: Template/PythonFastAPI/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Power Skill API
+
+Sample PowerSkill API
+
+## Solution Flow
```
Comment (bleroy) [inline Template/PythonFastAPI/README.md:5]: Why the empty section?

File: Template/PythonFastAPI/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+
+## Environment variables
+
+The following sample environment variables need to be set for the process to work:
+
+```bash
+
```
Comment (bleroy) [inline Template/PythonFastAPI/README.md:13]: You may remove the empty line and unnecessary indentation.

File: Template/PythonFastAPI/README.md (hunk 1 above)
Comment (bleroy) [inline Template/PythonFastAPI/README.md:5]: Why the empty section?

File: Template/PythonFastAPI/README.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        "correlationId": "123233434334334",
+        "batch": "4535346534654654",
+        "errors" : "",
+        "data" : {
+          "entities": [
+            {
+              "modelName" : "[modelname - e.g 1]",
+              "language": "AR",
+              "text" : [
+                 "ريك أستلي" ,"لورانس"
```
Comment (bleroy) [inline Template/PythonFastAPI/README.md:58]: Are you actually rickrolling us? :D

File: Template/PythonFastAPI/app.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+from powerskill import powerskill
+import os
+from starlette.status import HTTP_403_FORBIDDEN
+
+
+load_dotenv()
+app = FastAPI()
+
+class Values(BaseModel):
+    values:List = []
```
Comment (bleroy) [inline Template/PythonFastAPI/app.py:16]: nit: missing space

File: Template/PythonFastAPI/app.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+async def get_api_key(
+    api_key_header: str = Security(api_key_header),
+):
+
+    if api_key_header == API_KEY:
+        return api_key_header
+    else:
+        raise HTTPException(
+            status_code=HTTP_403_FORBIDDEN, detail="CogSvc Key not present"
```
Comment (bleroy) [inline Template/PythonFastAPI/app.py:36]: nit: missing spaces around `=`

File: Template/PythonFastAPI/README.md (hunk 3 above)
Comment (bleroy) [reply Template/PythonFastAPI/README.md:58]: Can we instead use the actual output from the sample as you'd get if you ran it as is?

**Review by bleroy — CHANGES_REQUESTED** [review]
You also have a couple of empty readme files that you can remove.

Thanks for the contribution!

File: Template/PythonFastAPI/README.md (hunk 1 above)
Comment (shanepeckham) [reply Template/PythonFastAPI/README.md:5]: Removed empty section

File: Template/PythonFastAPI/README.md (hunk 3 above)
Comment (shanepeckham) [reply Template/PythonFastAPI/README.md:58]: Implemented

**Comment by bleroy** [comment]
Thanks for the contribution.

**Pull request merged**